packages/svelte/src/internal/server/index.js

```diff
@@ -79,6 +96,58 @@ export function assign_payload(p1, p2) {
 	p1.anchor = p2.anchor;
 }
 
+/**
+ * @param {Payload} payload
+ * @param {string} message
+ */
+function error_on_client(payload, message) {
+	message =
+		`Svelte SSR validation error:\n\n${message}\n\n` +
+		'Ensure your components render valid HTML as the browser will try to repair invalid HTML, ' +
+		'which may result in content being shifted around and will likely result in a hydration mismatch.';
+	// eslint-disable-next-line no-console
+	console.error(message);
+	payload.head.out += `<script>console.error(${message})</script>`;
```

Comment:
Missing quotes around `${message}`?

Reply:
It’s already a string though?

Reply:
yes, but you're generating code (the script tag), and in code strings must be quoted...

Reply:
You're right, published a fix